validator/testcases/javascript/actions.py

```diff
@@ -81,16 +78,26 @@ def trace_member(traverser, node, instantiate=False):
         base = trace_member(traverser, node["object"], instantiate)
         base = _expand_globals(traverser, base)
 
+        identifier = _get_member_exp_property(traverser, node)
+
         # Handle the various global entity properties.
         if base.is_global:
             # If we've got an XPCOM wildcard, return a copy of the entity.
             if "xpcom_wildcard" in base.value:
                 traverser._debug("MEMBER_EXP>>XPCOM_WILDCARD")
+
+                from predefinedentities import CONTRACT_ENTITIES
+                if identifier in CONTRACT_ENTITIES:
+                    kw = dict(err_id=("js", "actions", "dangerous_contract"),
+                              warning="Dangerous XPCOM contract ID")
```

Comment:
Why not define all the dict items in the same place (either here or in `prefedinedentities`)?